unit_test/fuzzing/test_responder/test_spdm_responder_challenge_auth/challenge_auth.c

```diff
@@ -306,6 +402,13 @@ void libspdm_run_test_harness(void *test_buffer, size_t test_buffer_size)
     libspdm_test_responder_challenge_case6(&State);
     libspdm_unit_test_group_teardown(&State);
 
+    libspdm_unit_test_group_setup(&State);
+    libspdm_test_responder_challenge_case7(&State);
+    libspdm_unit_test_group_teardown(&State);
+
+    libspdm_unit_test_group_setup(&State);
+    libspdm_test_responder_challenge_case8(&State);
+    libspdm_unit_test_group_teardown(&State);
```

Comment:
can you add some comment for the new test case7 and case8 to describe its test scenario?